lib/lens.rb

```diff
@@ -1,4 +1,9 @@
 require "lens/core"
+if defined?(Rails)
+  require "lens/railtie"
+else
+  raise RuntimeError, "WTF!?!?! Where Rails man?"
```

Comment:
Redundant `RuntimeError` argument can be removed.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.